components/ingest-service/pipeline/publisher/node_manager_publisher.go

```diff
@@ -75,11 +75,14 @@ func gatherInfoForNode(node backend.Node) (*manager.NodeMetadata, error) {
 			Value: tag,
 		}
 	}
+	if node.Environment != "" {
+		tags = append(tags, &common.Kv{Key: "environment", Value: node.Environment})
+	}
 
 	return &manager.NodeMetadata{
 		Uuid:            node.EntityUuid,
 		Name:            node.NodeName,
-		PlatformName:    node.Platform,
+		PlatformName:    node.PlatformFamily,
```

Comment:
maybe we want to eventually go to just sending the platform full field?  for now, i switched it to using family for consistency with compliance side of things, since it's sending name/version as two sep fields

Reply:
For our example CCR https://github.com/chef/automate/blob/82772a8c0a82da983bb9b329fb31716447d66609/components/ingest-service/examples/converge-success-report.json#L1391 The platform_family "rhel" is different than the platform "redhat").

Reply:
I think I have a solution :thinking:

Reply:
Here is one solution https://github.com/chef/automate/pull/3612